### PR TITLE
[BPK-2732]: BpkNudger to support outline buttons

### PR DESCRIPTION
**Pull request by ojcurtis**
Remember to include the following changes:
+ [x] `UNRELEASED.md`
+ [x] `README.md`
+ [x] Tests
+ [x] Docs (either update [backpack-docs](https://github.com/Skyscanner/backpack-docs) now, or create a follow up ticket)

### CODE REVIEW

**Comment by ojcurtis** [comment]
Screenshot 2019-07-23 at 16 32 46

File: packages/bpk-component-nudger/README.md [hunk 1]
```diff
@@ -59,6 +59,7 @@ class App extends Component {
 | onChange              | func                          | true     | -             |
 | value                 | number                        | true     | -             |
 | className             | string                        | false    | null          |
+| buttonType            | string                        | false    | secondary     |
```
Comment (shaundon) [inline packages/bpk-component-nudger/README.md]: Could we change `string` to `oneOf('secondary', 'outline')`?

File: packages/bpk-component-nudger/src/BpkNudger.js [hunk 2]
```diff
@@ -106,10 +114,12 @@ BpkNudger.propTypes = {
   value: PropTypes.number.isRequired,
   onChange: PropTypes.func.isRequired,
   className: PropTypes.string,
+  buttonType: PropTypes.string,
```
Comment (shaundon) [inline packages/bpk-component-nudger/src/BpkNudger.js]:
```suggestion
  buttonType: PropTypes.oneOf(['secondary', 'default']),
```

Comment (ojcurtis) [reply packages/bpk-component-nudger/src/BpkNudger.js]: I would probably change it to `oneOf(['secondary', 'outline'])`, wdyt?

Comment (shaundon) [reply packages/bpk-component-nudger/src/BpkNudger.js]: Yeah sorry, I mean to write `outline` but wrote `default` for some reason 🤦‍♂

File: packages/bpk-component-nudger/README.md (hunk 1 above)
Comment (ojcurtis) [reply packages/bpk-component-nudger/README.md]: should be updated

File: packages/bpk-component-nudger/src/BpkNudger.js (hunk 2 above)
Comment (ojcurtis) [reply packages/bpk-component-nudger/src/BpkNudger.js]: should be updated 😄

**Pull request merged**